### PR TITLE
clarified upgrade mode per-user instead per-tenant

### PR DESCRIPTION
**Pull request by MartinRinas**
the upgrade mode of the users involved are relevant, not the upgrade mode of the entire tenant.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/native-chat-for-external-users.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ description: Learn about the native Teams chat experience for external access (f
...
 
-When a Microsoft Teams users is chatting with an external (federated) user, the chat experience is limited to text. However, if both your Teams tenant and that of the external user is in the TeamsOnly upgrade mode, you can have a "native-Teams chat experience," which includes rich formatting, @mentions, and other chat features. In other words, you can have the same rich 1:1 Teams chat experience with eligible external users as you'd have with users in your organization. Native Teams chats with external users are still limited to 1:1 chats only (external users can't do group chats).
+When a Microsoft Teams users is chatting with an external (federated) user, the chat experience is limited to text. However, if both your Teams user and the external user is in the TeamsOnly upgrade mode, you can have a "native-Teams chat experience," which includes rich formatting, @mentions, and other chat features. In other words, you can have the same rich 1:1 Teams chat experience with eligible external users as you'd have with users in your organization. Native Teams chats with external users are still limited to 1:1 chats only (external users can't do group chats).
```
Comment (JohanFreelancer9) [inline Teams/native-chat-for-external-users.md:23]:
```suggestion
When a Microsoft Teams user is chatting with an external (federated) user, the chat experience is limited to text. However, if both your Teams user and the external user is in the TeamsOnly upgrade mode, you can have a "native-Teams chat experience," which includes rich formatting, @mentions, and other chat features. In other words, you can have the same rich 1:1 Teams chat experience with eligible external users as you'd have with users in your organization. Native Teams chats with external users are still limited to 1:1 chats only (external users can't do group chats).
```

Comment (JohanFreelancer9) [reply Teams/native-chat-for-external-users.md:23]: Please correct "When a Microsoft Teams users is..." to "When a Microsoft Teams user is..."

File: Teams/native-chat-for-external-users.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ description: Learn about the native Teams chat experience for external access (f
...
 
-The native chat experience for external users is turned on for all Teams tenants, but not all users are eligible. To be offered a native chat experience, both the sender and receiver need to be on a Teams tenant that's running the TeamsOnly upgrade mode. To learn more about upgrade policies, read [Setting your coexistence and upgrade settings](setting-your-coexistence-and-upgrade-settings.md).
+The native chat experience for external users is turned on for all Teams tenants, but not all users are eligible. To be offered a native chat experience, both the sender and receiver need to be configure for TeamsOnly upgrade mode. To learn more about upgrade policies, read [Setting your coexistence and upgrade settings](setting-your-coexistence-and-upgrade-settings.md).
```
Comment (JohanFreelancer9) [inline Teams/native-chat-for-external-users.md:25]:
```suggestion
The native chat experience for external users is turned on for all Teams tenants, but not all users are eligible. To be offered a native chat experience, both the sender and receiver need to be configured for TeamsOnly upgrade mode. To learn more about upgrade policies, read [Setting your coexistence and upgrade settings](setting-your-coexistence-and-upgrade-settings.md).
```

Comment (JohanFreelancer9) [reply Teams/native-chat-for-external-users.md:25]: Please correct "configure" to "configured".

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**